### ISSUE TITLE
Favouriting Events: Saved event icons flicker after rotating device to landscape and back to portrait (iOS only)

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#223]
## Description of issue

When you are on the Events listing page, if you are viewing several events that have been "saved", then their heart icons will flicker on and off a few times (before returning to the correct state) when rotating the device to landscape and back to portrait.  

## Expected behaviour

The heart icons don't flicker when rotating the device. 

## Steps to reproduce

- Open the Events listing page
- Tap to favourite the top 3 events
- Rotate device to landscape
- Rotate it back to portrait
> Heart icons flicker off and on a few times before returning to correct state. 

## OS / Browser / Device model

iPhone 6 (10.3.3)
iPhone 7+ (10.3.1)

## Screenshots

Couldn't get a screen recording, but can provide a demonstration if needed.

**Comment by RGBboy** [comment]
Closing as this should be fixed by @282
